src/elf/dynamic.rs

```diff
@@ -1,8 +1,10 @@
+use crate::zerocopy;
```

Comment:
Zerocopy is still used here, and in many other places. Did you forget to `git push`?

Reply:
Yes, I have named the "one-file-inlined crate" as the original crate, so as to give credit where it is due 😅